### PR TITLE
minimOS: removed lua packages and its dependencies.

### PR DESCRIPTION
**Pull request by sagarjminim**
SW-2923

### CODE REVIEW

File: profiles/ucentral-ap.yml [hunk 1]
```diff
@@ -9,7 +9,6 @@ feeds:
 include:
   #- webui
   - openflow
-  - fbwifi
```
Comment (bmastbergen) [inline profiles/ucentral-ap.yml:12]: NIT, but can we comment it out, like we did for webui?

Comment (sagarjminim) [reply profiles/ucentral-ap.yml:12]: fine. I will do it.

Comment (bmastbergen) [reply profiles/ucentral-ap.yml:12]: more NIT.  #fbwifi should replace fbwifi.  It should be on the same line it was originally

Comment (sagarjminim) [reply profiles/ucentral-ap.yml:12]: @bmastbergen Done please review.

**Review by bmastbergen — APPROVED** [review]

**Pull request merged**